diff --git a/node_modules/@types/react-native/index.d.ts b/node_modules/@types/react-native/index.d.ts
index 1caa4c7..adb4497 100644
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -1084,6 +1084,11 @@ export interface TextProps extends TextPropsIOS, TextPropsAndroid, Accessibility
      * - >= 1: sets the maxFontSizeMultiplier of this node to this value
      */
     maxFontSizeMultiplier?: number | null | undefined;
+
+    /**
+     * Specifies href
+     */
+    href?: string;
 }
 
 /**
